package.json

```diff
@@ -48,8 +48,8 @@
     "@angular-devkit/core": "18.0.1",
     "@angular-devkit/schematics": "18.0.1",
     "@angular-eslint/builder": "17.3.0",
-    "@angular-eslint/eslint-plugin": "17.3.0",
-    "@angular-eslint/eslint-plugin-template": "17.3.0",
+    "@angular-eslint/eslint-plugin": "18.3.0",
+    "@angular-eslint/eslint-plugin-template": "18.3.0",
     "@angular-eslint/schematics": "17.5.1",
     "@angular-eslint/template-parser": "17.3.0",
```

Comment:
Can you also update the other `angular-eslint` packages please?
When it's updates we should  be a able to remove the `--force` here:

https://github.com/testing-library/angular-testing-library/blob/81b457b8e5ec8e15d8af92cfdfdb666e83f90046/.github/workflows/ci.yml#L36

Reply:
Done ✅